api/python/place_in.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
-title: Places within a Place
-nav_order: 10
+title: Places Within a Place
```

Comment:
please revert the capitalization of "within" for consistency (anne was careful to follow the google documentation style guide)